easybuild/easyconfigs/p/polymake/polymake-4.8-foss-2022a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'ConfigureMake'
+
+name = 'polymake'
+version = '4.8'
+
+homepage = 'https://polymake.org'
+description = """polymake is open source software for research in polyhedral geometry.
+It deals with polytopes, polyhedra and fans as well as simplicial complexes, matroids,
+graphs, tropical hypersurfaces, and other objects."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['https://polymake.org/lib/exe/fetch.php/download/']
+sources = [SOURCE_TAR_BZ2]
+checksums = ['3f7bd19d6ac1372ae32368e57134ff37a11fbe9eb1f6abe8195b34c5eec29b11']
+
+dependencies = [
+    ('Ninja', '1.10.2'),
+    ('libxml2', '2.9.13'),
+    ('libxslt', '1.1.34'),
+    ('libreadline', '8.1'),
```

Comment:
```suggestion
    ('libreadline', '8.1.2'),
```

Reply:
Resolved in https://github.com/easybuilders/easybuild-easyconfigs/pull/17212/commits/ab01bd22c6e71b2c9027ca807c8e3fdb859bb240